lib/gat/gatcaddyfile/pool.go

```diff
@@ -115,6 +116,26 @@ func init() {
 					} else {
 						module.ExtendedQuerySync = true
 					}
+				case "packet_tracing_option":
+					if d.NextArg() {
+						val, err := strconv.Atoi(d.Val())
```

Comment:
should probably just switch over Val and check for "client", "server", "both", or "disabled" and set it so we don't need magic constants in the caddyfile

Reply:
I can do that. I was slightly worried that a value of 3 would be a head scratcher if you weren't intimate with the option